drivers/timer/Kconfig

```diff
@@ -417,6 +417,18 @@ config MCUX_OS_TIMER
 	  This module implements a kernel device driver for the NXP OS
 	  event timer and provides the standard "system clock driver" interfaces.
 
+DT_NODELABEL_NXP_GPT_TIMER := gpt2
+
```

Comment:
This is limiting to `gpt2`, we should try to keep this generic in case future platforms have a different gpt